schema/gene_families.js

```diff
@@ -0,0 +1,11 @@
+import gravity from "../lib/loaders/gravity"
+import GeneFamily from "./gene_family"
+import { GraphQLList } from "graphql"
+
+const GeneFamilies = {
+  type: new GraphQLList(GeneFamily.type),
+  description: "A list of Gene Families",
+  resolve: () => gravity("gene_families"),
```

Comment:
Can you replace the direct use of the `gravity` loader with defining a specific loader [here](https://github.com/artsy/metaphysics/blob/master/lib/loaders/per_type.js)? Your resolve function would then be defined like so:

```js
resolve:  (_source, _args, _request, { rootValue: { geneFamiliesLoader } }) => geneFamiliesLoader(),
```